demisto_sdk/commands/validate/validators/RM_validators/RM107_is_template_sentence_in_readme.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Union
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+from demisto_sdk.commands.common.tools import search_substrings_by_line
+
+ContentTypes = Union[Integration, Script, Playbook, Pack]
+
+
+class IsTemplateInReadmeValidator(BaseValidator[ContentTypes]):
+    error_code = "RM107"
+    description = "Checks if there are the generic sentence '%%FILL HERE%%' in the README content."
+    rationale = "Checks if there are the generic sentence '%%FILL HERE%%' in the README content."
```

Comment:
```suggestion
    rationale = "Ensuring our documentation looks good and professional."
```
the goal of rationale is to explain the "why", rather than "what".